sdk/core/core-client/review/core-client.api.md

```diff
@@ -223,13 +223,16 @@ export interface OperationRequestInfo {
 
 // @public
 export interface OperationRequestOptions {
+    accept?: string;
     allowInsecureConnection?: boolean;
+    contentType?: string;
     customHeaders?: {
         [key: string]: string;
     };
     onDownloadProgress?: (progress: TransferProgressEvent) => void;
     onUploadProgress?: (progress: TransferProgressEvent) => void;
     shouldDeserialize?: boolean | ((response: PipelineResponse) => boolean);
+    skipEncoding?: boolean;
```

Comment:
Actually, there is a behavior mismatch between the currently skipUrlEncoding in the core-client and core-client-rest. 
The core-client has this `skipEncoding` option in both the PathParameter and QueryParameter, which means it can set only encode the pathParameter or only encode the queryParameters, but in RLC, we have only one option `skipUrlEncoding` once it's set, it will impact both the pathParameters and queryParameters.

Reply:
Interesting. I'd say leave the existing behavior for now unless we can think of a case where it's not desirable.

Reply:
added another option, see here https://github.com/Azure/azure-sdk-for-js/pull/26331#discussion_r1244963443